### ISSUE TITLE
Ambiguous use of 'cornerRadius'

### ISSUE THREAD

**Issue by Isuru-Nanayakkara** [SCENEE/FloatingPanel#152]
### Short description

I have a `UIView` extension created for `cornerRadius` property. It's conflicting with the `cornerRadius` property of the `surfaceView`.

### Expected behavior

Not receive a _Ambiguous use of 'cornerRadius'_ error.

### Actual behavior

Receives a _Ambiguous use of 'cornerRadius'_ error.

### Environment

**Library version**

1.4.0

**Installation method**

- [x] CocoaPods
- [ ] Carthage
- [ ] Git submodules

**iOS version(s)**

12.1

**Xcode version**

10.1

**Comment by SCENEE** [comment]
It looks the same issue as https://stackoverflow.com/questions/38326479/extension-property-conflicting-subclass-declaration-in-swift. Hmm..

**Comment by SCENEE** [comment]
I'm considering to let  `FloatingPanelSurfaceView.backgroundView` open to access from your app. 

```diff
--- a/Framework/Sources/FloatingPanelSurfaceView.swift
+++ b/Framework/Sources/FloatingPanelSurfaceView.swift
@@ -59,7 +59,7 @@ public class FloatingPanelSurfaceView: UIView {
     /// The color of the surface border.
     public var borderWidth: CGFloat = 0.0  { didSet { setNeedsLayout() } }

-    private var backgroundView: UIView!
+    public var backgroundView: UIView!
     private var backgroundHeightConstraint: NSLayoutConstraint!
```

The change will let you solve the error by adding the following `fp_cornerRadius` property and use it instead of `FloatingPanelSurfaceView.cornerRadius` directly that conflicts your UIView extension.

```swift
public extension FloatingPanelSurfaceView {
    @objc public dynamic var fp_cornerRadius: CGFloat {
        get { return backgroundView.layer.cornerRadius }
        set {
            contentView?.layer.cornerRadius = newValue
            backgroundView.layer.cornerRadius = newValue
        }
    }
}
```

**Comment by Isuru-Nanayakkara** [comment]
Thanks @SCENEE. This will be added to the next version of the library, I'm guessing?

**Comment by SCENEE** [comment]
That's right! This will be released in v1.4.1!

**Comment by atereshkov** [comment]
Same for me! Thanks for the fix!
